OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -386,6 +386,17 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					}
 				}
 
+				// Renamed AttackSequence to DefaultAttackSequence in WithInfantryBody.
+				if (engineVersion < 20160908)
```

Comment:
I think you can update this to something like 20161016 for now, maybe we'll get it merged quickly.